addon/-private/system/store.js

```diff
@@ -3411,11 +3410,7 @@ function getModelFactory(store, cache, normalizedModelName) {
 function _lookupModelFactory(store, normalizedModelName) {
   let owner = getOwner(store);
 
-  if (owner.factoryFor) {
-    return owner.factoryFor(`model:${normalizedModelName}`);
-  } else {
-    return owner._lookupFactory(`model:${normalizedModelName}`);
-  }
+  return owner.factoryFor(`model:${normalizedModelName}`);
```

Comment:
Just to confirm, this drops support for Ember < 2.12 (unless someone brings in the factoryFor polyfill): what is the current minimum Ember version supported by Ember Data?

Reply:
We have a "last two LTS" policy, meaning we support Ember 2.18+ currently